waspc/data/Generator/templates/server/src/routes/auth/passport/google/googleDefaults.js

```diff
@@ -0,0 +1,23 @@
+import { generateAvailableDictionaryUsername } from '../../../../core/auth.js'
+
+// Default implementation if there is no `auth.methods.google.configFn`.
+export function configFn() {
+  const clientId = process.env['GOOGLE_CLIENT_ID']
+  const clientSecret = process.env['GOOGLE_CLIENT_SECRET']
+
+  if (!clientId) {
+    throw new Error("Missing GOOGLE_CLIENT_ID environment variable.")
+  }
+
+  if (!clientSecret) {
+    throw new Error("Missing GOOGLE_CLIENT_SECRET environment variable.")
+  }
+
+  return { clientId, clientSecret, scope: ['profile'] }
+}
+
+// Default implementation if there is no `auth.methods.google.getUserFields`.
```

Comment:
We no longer have an `onSignInFn` here. Instead, we just return the user fields required to create a new user.